config/clusters/templates/common/common-hub.values.yaml

```diff
@@ -0,0 +1,58 @@
+
+#If the hub is a daskhub, please indent everything under a basehub key
+#basehub:
```

Comment:
Will need to make the command also accept `hub_type` as it doesn't at the moment and I'm not sure we should, since it's a super small change to make.

Also, since this command outputs the config to stdout, we'll be compelled to verify the config when we paste it in the file and do the indent manually anyway and the indentation will still need to be done manually.

I'd say to leave it like this for now and if it proves to be an error generator, we'll introduce hub_type and the validation failure. WDYT?